web/src/features/fba/components/AboutDataPopover.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import Popover from '@mui/material/Popover'
+import Typography from '@mui/material/Typography'
+import InfoIcon from '@mui/icons-material/Info'
+import { theme } from 'app/theme'
+
+interface AboutDataProps {
+  advisoryThreshold: number
+}
+
+const AboutDataPopover = ({ advisoryThreshold }: AboutDataProps) => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null)
+
+  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null)
+  }
+
+  const open = Boolean(anchorEl)
+
+  return (
+    <div data-testid="about-data-popover">
+      <Typography
+        data-testid="about-data-trigger"
+        fontSize={'0.75rem'}
+        onClick={handlePopoverOpen}
+        sx={{ cursor: 'pointer', display: 'flex', alignItems: 'center' }}
+      >
+        <InfoIcon sx={{ marginRight: theme.spacing(1), color: '#4681f4' }}></InfoIcon>
+        About this data
+      </Typography>
+      <Popover
+        id="about-data-popover"
+        open={open}
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left'
+        }}
+        onClose={handlePopoverClose}
+        disableRestoreFocus
+        slotProps={{ paper: { sx: { maxWidth: 350, backgroundColor: '#EEEEEE' } } }}
```

Comment:
Is this #EEEEEE color used anywhere else in the app? If so, perhaps we could import it? If not, I wonder if we should match a pre-existing color (eg. the background around the advisory bulletin box perhaps)?

Reply:
Thank you!